src/lib/components/layout/v2/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { ReactElement } from 'react';
+import styled from 'styled-components';
+import * as defaultTheme from '../../../style/theme';
+import { getTheme } from '../../../utils';
+
+const LayoutContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  box-sizing: border-box;
+  background: ${(props) => getTheme(props)['backgroundLevel1']};
+`;
+
+const Navigation = styled.div`
+  height: ${defaultTheme.navbarHeight};
+`;
+
+export const ContextContainer = styled.div<{
+  background?: string;
+}>`
+  background: ${(props) => props.background || 'initial'};
+  height: 3rem;
+  padding: 1rem;
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+`;
+
+export const OverallSummary = styled.div`
```

Comment:
I think we should have a padding on this OverallSummary component.

Reply:
To be precise only left-right padding